### PR TITLE
Clarification to condition for greater or lesser than

### PR DESCRIPTION
**Pull request by frheault**
In relation to #385 . This add new condition to image_math and connectivity math
- lower_threshold -> lower_threshold_eq
- upper_threshold -> upper_threshold_eq
The >= and <= are not clearly distinct.

The script scil_filter_connectivity was unclear. The greater_than and lesser_than option were actually greater_or_equal and lesser_or_equal. To make it clearer, while keeping the script easy to read (and simple) the equals were removed and now the script does what the paramters are saying. Strictly greater or lesser than.

@mdesco This should be slightly clearer

### CODE REVIEW

File: scilpy/image/operations.py [hunk 1]
```diff
@@ -121,6 +123,43 @@ def _validate_float(x):
         raise ValueError
 
 
+def lower_threshold_eq(input_list, ref_img):
+    """
+    lower_threshold_eq: IMG THRESHOLD
+        All values below the threshold will be set to zero.
+        All values above (or equal) the threshold will be set to one.
```
Comment (mdesco) [inline scilpy/image/operations.py:130]: remove () please
-> All values above or equal to the threshold ...

File: scilpy/image/operations.py [hunk 2]
```diff
@@ -121,6 +123,43 @@ def _validate_float(x):
...
+    output_data[data < input_list[1]] = 0
+    output_data[data >= input_list[1]] = 1
+
+    return output_data
+
+
+def upper_threshold_eq(input_list, ref_img):
+    """
+    upper_threshold_eq: IMG THRESHOLD
+        All values below (or equal) the threshold will be set to one.
```
Comment (mdesco) [inline scilpy/image/operations.py:147]: as above please. remove ()

**Review by mdesco — APPROVED** [review]
Minor edits and all good after.

File: scilpy/image/operations.py [hunk 3]
```diff
@@ -134,7 +173,7 @@ def lower_threshold(input_list, ref_img):
     output_data = np.zeros(ref_img.header.get_data_shape(), dtype=np.float64)
     data = input_list[0].get_fdata(dtype=np.float64)
     output_data[data < input_list[1]] = 0
```
Comment (arnaudbore) [inline scilpy/image/operations.py:175]: why the equal does not end up here ? same for upper_threshold

**Review by arnaudbore — APPROVED** [review]
Small comments, now you can validate commit suggested.

**Pull request merged**